nntrainer/layers/common_properties.h

```diff
@@ -30,20 +30,21 @@ namespace nntrainer {
  * accordingly
  */
 enum class ActivationType {
-  ACT_TANH,       /**< tanh */
-  ACT_SIGMOID,    /**< sigmoid */
-  ACT_RELU,       /**< ReLU */
-  ACT_SWISH,      /**< Swish */
-  ACT_GELU,       /**< GELU */
-  ACT_QUICK_GELU, /**< Quick GELU */
-  ACT_SOFTMAX,    /**< softmax */
-  ACT_SOFTPLUS,   /**< softplus */
-  ACT_LEAKY_RELU, /**< Leaky ReLU */
-  ACT_ELU,        /**< ELU */
-  ACT_SELU,       /**< SELU */
-  ACT_MISH,       /**< Mish */
-  ACT_NONE,       /**< no op */
-  ACT_UNKNOWN     /**< unknown */
+  ACT_TANH,         /**< tanh */
```

Comment:
There is a conflict. I think it will be fine if you just fix this.

I added pr 2665 which adds the missing activation type after this pr.

Reply:
Thanks. I rebased it.